packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -110,37 +146,57 @@ export function cloudflare<
 								type: 'ESModule',
 								path: runnerPath,
 							},
+							{
+								// we declare the workerd-custom-import as a CommonJS module, thanks to this
+								// require is made available in the module and we are able to handle cjs imports, etc...
+								type: 'CommonJS',
+								path: fileURLToPath(
+									new URL('workerd-custom-import.cjs', import.meta.url),
```

Comment:
Let's put this path in a variable at the top of the file with the others.

Reply:
done 👍